noir/noir-repo/compiler/noirc_frontend/src/node_interner.rs

```diff
@@ -623,6 +623,15 @@ impl NodeInterner {
         f(&mut value);
     }
 
+    pub fn update_struct_attributes(
```

Comment:
Not sure if we need some noir people to allow us here @TomAFrench, we are asking for your blessing 🙏

Reply:
Seems fine but I'd like to get https://github.com/AztecProtocol/aztec-packages/pull/7134 merged before making more modifications to the compile/aztec_macros

Reply:
ah, well. Nevermind.

Reply:
Nah, dw about it. Looks like I was a bit overly cautious as the sync seems unaffected.